tests/std/tests/P1951R1_default_arguments_pair_forward_ctor/test.compile.pass.cpp

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <memory>
+#include <utility>
+
+using namespace std;
+
+void test() {
+    pair<int, unique_ptr<int>> p{42, {}};

```

Comment:
Should we have another test case that uses the default template argument for the first template parameter?

Reply:
Oops - this started as "Should we have", and I'd intended to change it to "We should have", but apparently didn't make it all the way there.